.storybook/storybook.css

```diff
@@ -356,3 +356,13 @@ input[type="color"]::-webkit-color-swatch {
 	align-items: center;
 	flex: 1 0 auto;
 }
+
+.f-div-highlight-overlay {
```

Comment:
storybook.css is specific for stroybook then why we added this here? we can add this in `f-root.scss`